src/components/social/facebook/index.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-shadow */
```

Comment:
@Manorlds-Eaglespark we can do away with these comments in the code. Alternatively, if you need to customize the linting configuration, do so in the `.eslintrc.json` file

Reply:
Done, kindly review again.